scripts/SplitStockholm.jl

```diff
@@ -35,26 +39,39 @@ end
 const Args = parse_commandline()
 
 function main(input)
-	infh = GZip.open(input)
-	lines = []
-	id = "no_accessionumber"
-	for line in eachline(infh)
-		if length(line) > 7 && line[1:7] == "#=GF AC"
-			id = get_n_words(line, 3)[3]
-		end
-		push!(lines, line)
-		if line == "//\n"
-			filename = joinpath(Args["path"], string(id, ".gz"))
-			outfh = GZip.open(filename, "w")
-			for l in lines
-				write(outfh, string(l))
-			end
-			close(outfh)
-			id = "no_accessionumber"
-			empty!(lines)
-		end
-	end
-	close(infh)
+    infh = GZip.open(input)
+    lines = []
+    id = "no_accessionumber"
+
+    if Args["progress"]
+        totalsize = filesize(input)
+        val = 0
+        prog = Progress(totalsize, 1)
+    end
+
+    # for line in eachline(infh)
+    while !eof(infh)
+        line = readline(infh)
+        if length(line) > 7 && line[1:7] == "#=GF AC"
+            id = get_n_words(line, 3)[3]
```

Comment:
The last version of `get_n_words` doesn't chomp the line, so we need to change this code line with `id = get_n_words(String(chomp(line)), 3)[3]`